README.md

```diff
@@ -209,10 +210,8 @@ enum_test.py: error: argument --food: invalid choice: 'eggs' (choose from ['ham'
 
 **NOTE**: enums are passed by name on the command line and not by value.
 
-- compatibility with both `dataclass` and `attrs`
```

Comment:
Why is this removed?

Reply:
This line and the line below didn't make much sense to me here, so I assumed they were left by mistake.

Compatibility of the library with both dataclasses/attrs is already established above, as well as support for `field` and `attr.ib` arguments. Not sure why they'd be bullet points in a section about enums.